style.css

```diff
@@ -121,3 +164,214 @@ body {
   margin-top: 10px;
   flex-grow: 1;
 }
+
+.grid-works {
+  display: grid;
+  grid-template-columns: 320px;
+  grid-template-rows: 100px 342px 342px 342px 342px 342px 342px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
```

Comment:
Let's not use a fixed width because this is causing the scaling issue. If you inspect this app on different mobile screens, I am sure you will understand.